DirectEmployers/MyJobs
# Pd 3616 user mgmt navigation

## What's this PR do?
- Moves navigation for DE 'Manage Users' Application to a top Sub Nav.
- Moves task actions to the top of the page. (ie: Add User)

## Screenshots (if appropriate)
<img width="1039" alt="screen shot 2017-08-24 at 11 55 17 am" src="https://user-images.githubusercontent.com/25237191/29675589-97010ff0-88c3-11e7-9215-0da40d12f327.png">

<img width="398" alt="screen shot 2017-08-24 at 12 18 39 pm" src="https://user-images.githubusercontent.com/25237191/29676716-c99e8fac-88c6-11e7-95e6-f62b71d0a528.png">


## Where should the reviewer start?
- Apps > Manage Users

## How should this be manually tested?
- Each task in the Sub Nav should change the view accordingly when clicked 
- The tasks 'Users' and 'Role' should toggle to the actions, 'Add User' and 'Add Role'
- Actions should also appear in top right corner of the contents container 

Mobile
- Tasks are shifted to a dropdown
- Clicking a task in the Sub Nav dropdown should change the view accordingly
